functests/2_performance_update/updating_profile.go

```diff
@@ -195,26 +194,47 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		table.DescribeTable("Verify that profile parameters were updated", func(cmd, parameter []string, shouldContain bool, useRegex bool) {
 			for _, node := range workerRTNodes {
 				for _, param := range parameter {
-					result, err := nodes.ExecCommandOnNode(cmd, &node)
-					Expect(err).ToNot(HaveOccurred(), "failed to execute %s", cmd)
-
-					matcher := ContainSubstring(param)
-					if useRegex {
-						matcher = MatchRegexp(param)
-					}
-
-					if shouldContain {
-						Expect(result).To(matcher)
+					if strings.ContainsAny(cmd[0], "1G | 2M") {
```

Comment:
It's better, but I'm still not too happy about this special case

Reply:
okay, actually since this case passes through a function which is common to other test cases where it accepts slice. I used this approach.

Reply:
Sure, makes sense. But still, let's try to improve. What about changing like this: https://gist.github.com/fromanirh/2e1ae827199c147c52b7431845d3cd50 (to be applied on top of your changes) ?

Reply:
warning: untested (but compiles fine :) )

Reply:
testing now

Reply:
@fromanirh  i tested your changes. It worked. I have update the patch with your changes.